lang/python/django-appconf/Makefile

```diff
@@ -15,7 +15,6 @@ PKG_LICENSE:=BSD-3-Clause
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://pypi.python.org/packages/34/b9/d07195652ab494b026f7cb0341dd6e5f2e6e39be177abe05e2cec8bd46e4/
 PKG_HASH:=6a4d9aea683b4c224d97ab8ee11ad2d29a37072c0c6c509896dd9857466fb261
-PKG_BUILD_DEPENDS:=python python-setuptools
```

Comment:
fwiw: i was meaning to remove `python-setuptools` since, all that is required for the _BUILD_DEPENDS is just `python/host` which should be selected via the package's `DEPENDS:=+python`